src/common/harvest/harvest.js

```diff
@@ -244,7 +244,14 @@ export class Harvest extends SharedContext {
       }
       return Object.entries(input || {})
         .reduce((accumulator, [key, value]) => {
-          if (value !== null && value !== undefined && value.toString()?.length) {
+          if (
+            value !== null &&
+            value !== undefined && (
+              (typeof value === 'number') ||
+              (typeof value === 'string' && value.length > 0) ||
+              (Object.entries(value).length > 0)
+            )
+          ) {
```

Comment:
```suggestion
          if ((typeof value === 'number') ||
              (typeof value === 'string' && value.length > 0) ||
              (typeof value === 'object' && Object.keys(value).length > 0)) {
```

Reply:
`typeof null === 'object'` ;)

Reply:
more of a philosophical question here... but **should** payloads that are nullish even get to the harvest?  Seems like it would make more sense to filter those out at the point of creation (the feature).

Reply:
if features have nothing to harvest, they shouldn't send anything to the harvest module

Reply:
i mean yes, but this also handles the case where payloads are valid but some fields in the payload have empty strings or null or whatever.  But should that be handled in the feature?